ai2thor_wrapper/config.ini

```diff
@@ -0,0 +1,5 @@
+[ENV_SPECIFIC]
```

Comment:
Change config.ini to config.ini.example or something like that, then never commit your config to the main repo. Add to gitignore, this one should be only a clean guide for people starting on the repo

Reply:
done